ConsoleView.cpp

```diff
@@ -1,27 +1,57 @@
+#include "ConsoleView.h"
+#include "UMLRelationship.h"
 //All these functions return void, they are simply here 
 //for the user to call when they want to get information on 
 //a CLASS (or a list of all classes), RELATIONSHIP, or ATTRIBUTE
 
+
```

Comment:
My suggestion with the overloading the ostream operator is simply, don't. In fact, I'm not even sure with the way this will set up that it will run properly. It doesn't know how to print out a relationship object, and honestly it will just make it more confusing to someone else trying to read the code